src/main/java/com/aws/greengrass/mqtt/bridge/auth/MQTTClientKeyStore.java

```diff
@@ -135,16 +139,8 @@ public void updateCA(List<String> caCerts) throws IOException, CertificateExcept
             X509Certificate caCert = pemToX509Certificate(caCerts.get(i));
             keyStore.setCertificateEntry("CA" + i, caCert);
         }
-        setCaCerts(caCerts);
```

Comment:
why aren't we storing the ca cert anymore?

Reply:
should be there in the line below, just replaced setter w/ normal assignment

Reply:
hmm github may be reporting it weirdly

Reply:
yeah it's there it's weird because the line was part of setCaCerts